mods/cnc/weapons/missiles.yaml

```diff
@@ -197,62 +137,40 @@ MammothMissiles:
 			Wood: 60
 			Light: 100
 			Heavy: 50
-		DamageTypes: Prone50Percent, TriggerProne, ExplosionDeath
-	Warhead@2Smu: LeaveSmudge
-		SmudgeType: Crater
-		InvalidTargets: Vehicle, Structure, Wall, Husk, Trees, Creep
 	Warhead@3Eff: CreateEffect
 		Explosions: med_frag
 		ImpactSounds: xplos.aud
 
 227mm.stnk:
+	Inherits: ^MissileWeapon
```

Comment:
This again introduces a `MinRange: 0c512`, which again is probably a good thing.